pyproject.toml

```diff
@@ -18,6 +18,7 @@ indexclient = "^2.1.0"
 httpx = "^0.12.1"
 authutils = "^5.0.4"
 cdislogging = "^1.0.0"
+parsimonious = "0.8.1"
```

Comment:
```suggestion
parsimonious = "^0.8.1"
```

Reply:
I think it might be better to [pin the version](https://github.com/erikrose/parsimonious#status) in this case.

"Note that there may be API changes until we get to 1.0, so be sure to pin to the version you're using."

Reply:
i see, they don't follow semantic versioning rules 😅